developer_setup/running_test_suites.md

```diff
@@ -0,0 +1,57 @@
+## Running the test suites
+
+ManageIQ specs are split into several components:
+
+* **vmdb**
+* **automation**
+* **migrations**
+* **replication**
+* **javascript**
+* **self_service**
+* **brakeman**
+
+Each suite has a pair of Rake tasks to setup and run them, in the form of `test:<type>` and
+`test:<type>:setup` (if required):
+
+```bash
+rake test:vmdb                   # Run all specs except migrations, replication, and automation
+rake test:vmdb:setup             # Setup environment for vmdb specs
```

Comment:
would it make more sense to list the setup before the running step?
(although, I guess alphabetical makes sense to me too)